apps/console/src/features/users/api/users.ts

```diff
@@ -160,6 +160,32 @@ export const addUser = (data: UserDetailsInterface): Promise<any> => {
         });
 };
 
+/**
+ * Bulk add users.
+ * 
+ * @param data - SCIM2.0 compliant request body
+ * @returns a promise containing the response.
+ */
+export const addBulkUsers = (data: SCIMBulkEndpointInterface): Promise<any> => {
+    const requestConfig: RequestConfigInterface = {
+        data,
+        headers: {
+            "Access-Control-Allow-Origin": store.getState().config.deployment.clientHost,
```

Comment:
There is no use of sending CORS in client requests. CORS headers are meant to be set by server. 

Hence, let's remove this.

Reply:
Removed in https://github.com/wso2/identity-apps/pull/6597.